arclightning_backend/.gitignore

```diff
@@ -0,0 +1,4 @@
+/target
+**/*.rs.bk
+**/*.swp
```

Comment:
We can remove this from the backend gitignore and add it to the `.gitignore` for the entire repo. That way the frontend people can use vim without fear :joy:

Reply:
So I actually have it as you said, where there's a `.gitignore` for the entire repo.  For some reason it's not included in my PR?

Reply:
You have to git add the `.gitignore`